### PR TITLE
In test framework, process_transaction takes int as parameter instead of bytes

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: test/python/apps/ethereum.py [hunk 1]
```diff
@@ -80,8 +80,11 @@ def sign(self, extra_payload: bytes = bytes.fromhex('eb')):
         return self._exchange(Command.SIGN, payload=payload)
 
 
+def eth_amount_to_wei(eth_amount: int) -> int:
+    return round(eth_amount * 10**18)
+
 def eth_amount_to_wei_hex_string(eth_amount: int) -> str:
-    hex:str = '{:x}'.format(round(eth_amount * 10**18))
+    hex:str = '{:x}'.format(eth_amount_to_wei(eth_amount))
```
Comment (lpascal-ledger) [inline test/python/apps/ethereum.py:87]: `:str` is superfluous, no?

File: test/python/apps/exchange.py [hunk 2]
```diff
@@ -91,6 +91,11 @@ def _craft_conf_from_ticker(ticker_id: str, signer: SigningAuthority) -> bytes:
     derivation_path = TICKER_ID_TO_PACKED_DERIVATION_PATH[ticker_id]
     return prefix_with_len(currency_conf) + signed_conf + prefix_with_len(derivation_path)
 
+def _int_to_bytes(n: int) -> bytes:
+    if n == 0:
+        return b'\0'
+    else:
+        return n.to_bytes((n.bit_length() + 7) // 8, 'big')
```
Comment (lpascal-ledger) [inline test/python/apps/exchange.py:98]:
```suggestion
    return n.to_bytes(max(1, (n.bit_length() + 7) // 8), 'big')
```

Comment (lpascal-ledger) [inline test/python/apps/exchange.py:98]:
```suggestion
    return n.to_bytes((n.bit_length() + 7) // 8, 'big') or b'\0'  # case n is 0
```

**Review by lpascal-ledger — APPROVED** [review]

File: test/python/apps/ethereum.py (hunk 1 above)
Comment (fbeutin-ledger) [reply test/python/apps/ethereum.py:87]: Removed

**Pull request merged**